linetools/spectra/io.py

```diff
@@ -538,6 +542,57 @@ def parse_linetools_spectrum_format(hdulist):
 
     return xspec1d
 
+def parse_hdf5(inp, **kwargs):
+    """ Read a spectrum from HDF5 written in XSpectrum1D format
+    Expects:  meta, data, units
+
+    Parameters
+    ----------
+    inp : str or hdf5
+
+    Returns
+    -------
+
+    """
+    import json
+    import h5py
+    # Open
+    if isinstance(inp, basestring):
+        hdf5 = h5py.File(inp, 'r')
+    else:
+        hdf5 = inp
+    # Data
+    data = hdf5['data'].value
+    # Meta
+    if 'meta' in hdf5.keys():
```

Comment:
Check out the write_to_hdf5 method in XSpectrum1D
and the meta_to_disk mtehod in spectra.utils.

This has gotten a bit complex to handle multiple header
files for multiple spectra.